website/app/components/doc/page/cover.js

```diff
@@ -5,6 +5,10 @@ const STATUSES = {
     badgeType: 'success',
     badgeText: 'Released',
   },
+  comingSoon: {
+    badgeType: 'information',
+    badgeText: 'Coming soon',
+  },
```

Comment:
Now that you're not using anymore the badge, but the description, this can be removed.
```suggestion
```